src/steps/10-FinancialDetails/AppropriationOfFunds.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <v-container class="container-max-width mb-7" fluid>
+    <v-row>
+      <v-col class="col-12">
+        <h1 class="page-header">Let’s find out more about your appropriation of funds</h1>
```

Comment:
remove class `page-header` - that's used when no intro paragraph. Add class `mb-3` to the `h1`

Reply:
done